src/libmongoc/src/mongoc/mongoc-log.c

```diff
@@ -205,9 +205,9 @@ mongoc_log_default_handler (mongoc_log_level_t log_level, const char *log_domain
 #endif
 
    fprintf (stream,
-            "%s.%04ld: [%5d]: %8s: %12s: %s\n",
+            "%s.%04" PRId64 ": [%5d]: %8s: %12s: %s\n",
             nowstr,
-            tv.tv_usec / 1000L,
+            (int64_t)(tv.tv_usec / 1000L),
```

Comment:
```suggestion
            (int64_t) (tv.tv_usec / 1000),
```
ClangFormat + removal of redundant literal suffix.